custom/src/main/java/com/splunk/opentelemetry/jaeger/JaegerThriftSpanExporterFactory.java

```diff
@@ -16,23 +16,40 @@
 
 package com.splunk.opentelemetry.jaeger;
 
+import io.jaegertracing.thrift.internal.senders.HttpSender;
 import io.opentelemetry.exporter.jaeger.thrift.JaegerThriftSpanExporter;
 import io.opentelemetry.javaagent.spi.exporter.SpanExporterFactory;
 import io.opentelemetry.sdk.trace.export.SpanExporter;
 import java.util.Collections;
 import java.util.Properties;
 import java.util.Set;
+import okhttp3.OkHttpClient;
 
-// TODO remove when https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/1609
-// is implemented
 public class JaegerThriftSpanExporterFactory implements SpanExporterFactory {
```

Comment:
Shouldn't we test this (new) behaviour as well? To be sure that everything works properly with ant without token (ie no exception is thrown if not provided).

Reply:
It's not easy to unit test produced `SpanExporter` - this can be done, but either through checking fields via reflection or changing every constructor call to `Function<?, ?>` to stub that. I don't think that's worth it, and we have smoke tests anyway.